src/main/java/it/gov/pagopa/reward/notification/controller/NotificationControllerImpl.java

```diff
@@ -84,6 +89,28 @@ public Mono<Page<RewardExportsDTO>> getExportsPaged(String organizationId, Strin
                 .switchIfEmpty(Mono.just(Page.empty(pageable)));
     }
 
+    @Override
+    public Mono<ExportSummaryDTO> getSingleExportSummary(String organizationId, String initiativeId, String exportId) {
+        return exportDetailService.getExportSummary(organizationId, initiativeId, exportId);
```

Comment:
what if empty? I don't remember what's the behaviour, but let's test and check ;)